containers/src/Kokkos_StaticCrsGraph.hpp

```diff
@@ -21,6 +21,22 @@
 #define KOKKOS_IMPL_PUBLIC_INCLUDE_NOTDEFINED_STATICCRSGRAPH
 #endif
 
+#include <Kokkos_Macros.hpp>
+
+#if defined(KOKKOS_ENABLE_DEPRECATED_CODE_4)
+#if defined(KOKKOS_ENABLE_DEPRECATION_WARNINGS)
+namespace {
+[[deprecated("Deprecated <Kokkos_StaticCrsGraph.hpp> header is included")]] int
```

Comment:
Why not use `KOKKOS_DEPRECATED_WITH_COMMENT()` macro if we are including the macros?

Reply:
The symbol should always be deprecated when it's defined at all.